tests/qiskit_convert_test.py

```diff
@@ -267,17 +269,14 @@ def test_tketpass() -> None:
     assert np.allclose(u1, u2)
 
 
-def test_tketautopass() -> None:
+@pytest.mark.skipif(skip_remote_tests, reason=REASON)
```

Comment:
should we add a no timeout mark to this test as well? Have you thought about that?

Reply:
I will try without a timeout on this one as well.